examples/src/main/java/alluxio/examples/S3ASpecificOperations.java

```diff
@@ -0,0 +1,159 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.examples;
+
+import alluxio.Constants;
+import alluxio.cli.UnderFileSystemContractTest.RelatedS3Operations;
```

Comment:
nit: should we move the class `RelatedS3Operations` under `alluxio.examples`?

Reply:
Make sense. The import does not look good.
Moved to `alluxio.examples`